CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # nf-core/methylseq
 
+## [v1.4.1](https://github.com/nf-core/methylseq/releases/tag/1.4.1) - 2019-12-05
```

Comment:
```suggestion
## [v1.4.1](https://github.com/nf-core/methylseq/releases/tag/1.4.1) - 2019-12-06
```

Reply:
Done in https://github.com/nf-core/methylseq/pull/137/